apps/crn-server/src/config.ts

```diff
@@ -78,8 +79,8 @@ export const userInviteReturn = EMAIL_RETURN || 'hub.invites.dev@asap.science';
 export const eventBus = EVENT_BUS || 'asap-events-dev';
 export const eventSource = EVENT_SOURCE || '';
 export const eventBridgeEndpoint = EVENT_BRIDGE_ENDPOINT;
-export const eventBridgeAccessKey = EVENT_BRIDGE_ACCESS_KEY;
-export const eventBridgeSecret = EVENT_BRIDGE_SECRET;
+export const eventBridgeAccessKey = EVENT_BRIDGE_ACCESS_KEY!;
```

Comment:
actually you might want to consider using the `|| ''` pattern we used for all other env vars, sorry